r/R/feather.R

```diff
@@ -125,7 +125,8 @@ write_feather <- function(x,
 #'
 #' @inheritParams read_ipc_stream
 #' @inheritParams read_delim_arrow
-#' @param ... additional parameters, passed to [FeatherReader$create()][FeatherReader]
+#' @param ... additional parameters, passed to the internal function
+#' `make_readable_file()`
```

Comment:
It's internal but it's [documented](https://arrow.apache.org/docs/r/reference/make_readable_file.html) so we can link to it: 

```suggestion
#' [make_readable_file()].
```